parts/k8s/windowskubeletfunc.ps1

```diff
@@ -316,9 +316,14 @@ New-NSSMService {
         [string]
         [Parameter(Mandatory = $true)]
         $KubeProxyStartFile

```

Comment:
$KubeProxyStartFile
 =>
$KubeProxyStartFile,

Reply:
good catch. I believe this is why the last test run failed to provision.